articles/cdr-timescale-accounting/index.md

```diff
@@ -0,0 +1,166 @@
+
+### Carbon cycle feedbacks
+
+Regardless of whether or not a CDR approach exhibits delays to removal, global carbon cycle feedbacks affect the net atmospheric response to the carbon removal activity over time. This is because the global carbon cycle is a collection of interconnected reservoirs – the atmosphere, ocean, land, and geosphere – that are continually passing carbon back and forth. Whenever we add CO₂ to the atmosphere, or take it out, the rest of the carbon cycle responds.
+
+For example, burning fossil fuels moves carbon from the geosphere into the atmosphere. However, the CO₂ we emit into the atmosphere doesn’t all stay there. As the atmosphere equilibrates with other carbon reservoirs, a large share of emitted CO₂ is taken up by the land and surface ocean over the next decades to centuries. Over longer timescales (i.e., 1,000+ years), about a quarter of fossil CO₂ emissions remain in the atmosphere.<Cite id='joos.2013'/>
+
+Likewise, when a CDR project removes carbon from the atmosphere, an analogous process occurs. Over time, the reduction in atmospheric CO₂ concentrations that results from CDR is compensated for by natural outgassing of CO₂ from ocean and land carbon reservoirs. These carbon cycle feedbacks occur across all CDR approaches, with their magnitude depending on the rate of ongoing emissions, the atmospheric CO₂ concentration, and global mean temperature.<Cite id='ipcc.ar6.chp5'/>
+
+In Figure 1, we showed results from a model of DAC and DOR that ignored global carbon cycle feedbacks. In Figure 2, we show how carbon cycle feedbacks affect the net atmospheric response to DOR and DAC by running Hector with a coupled carbon cycle. For context, we also show the net atmospheric response to an emission pulse.
+
+<Figure>
+  <TimeSeries windSpeed='default' timeHorizon={100} emissions />
+  <FigureCaption number={2}>
+    Net atmospheric CO₂ response, including carbon cycle feedbacks, of removing
+    1 GtCO₂ from the atmosphere via <Purple>direct air capture</Purple> versus
+    removing 1 GtCO₂ from the ocean via <Pink>direct ocean removal</Pink>. By
+    comparison, 1 GtCO₂ of <Grey>fossil emissions</Grey> would trigger carbon
```

Comment:
small thing: the table is using `secondary` instead of `grey` for emissions -- possible to either update to `Secondary` here or update to `grey` in `TimeSeries`?